psakefile.ps1

```diff
@@ -103,9 +113,15 @@ task Bump {
   exec { npm version --no-git-tag-version $packageVersion }
 }
 
+task Npm -precondition { !(Test-Path (Join-Path $PSScriptRoot 'node_modules')) }{
+  exec { npm install }
+}
+
 task Vendor -depends VendorEditorServices, VendorEditorSyntax, VendorCytoscape
 
-task Build -depends Clean, Vendor, CompileTypeScript
+task BuildOrig -depends Clean, Npm, Vendor, CompileTypeScript
+
+task Build -depends Npm, Vendor, CompileTypeScript
 
 task Initial -depends Clean, Vendor
```

Comment:
Do we need to keep Initial given that it's not called in project.json anymore?

Reply:
Thanks. I had kept it during testing with intention to remove